doc/userguide/partials/options.rst

```diff
@@ -12,6 +12,14 @@
 
    Path to configuration file.
 
+.. option:: --include <path>
+
+   Additional configuration files to include. Multiple additonal
+   configuration files can be provided and will be included in the
+   order specified on the command line.  These additonal configuration
```

Comment:
nit: remove extra space before 'These'

Reply:
Can we add an example line showing how it would be done if one wanted to add multiple configuration files?